charts/kube-master/templates/secrets.yaml

```diff
@@ -18,9 +18,9 @@ data:
   openstack-router-id: {{ required "missing openstack-router-id" .Values.openstack.routerID | b64enc }}
   openstack-project-id: {{ required "missing openstack-project-id" .Values.openstack.projectID | b64enc }}
   openstack-region: {{ required "missing openstack-region" .Values.openstack.region | b64enc }}
-
   bootstrapToken: {{ required "missing boostrapToken" .Values.bootstrapToken | b64enc }}
   token.csv: {{ include (print $.Template.BasePath "/_token.csv.tpl") . | b64enc }}
+  node-password: {{ required "missing node-password" .Values.nodePassword | b64enc }}
```

Comment:
Could you also update the chart version to `master-1.9.0-kubernikus.1`? Even though this is informational only, we can see why Kluster is using which chart version.

```
🐢  > helm ls                                                                                                                                                                                                                                                                                                         0 ↵
NAME                                  	REVISION	UPDATED                 	STATUS  	CHART                         	NAMESPACE
test2-8b25871959204ff1a27605b7bcf873f7	1       	Tue Jan  9 11:34:00 2018	DEPLOYED	kube-master-1.9.0-kubernikus.0	kubernikus
v6-5d725ddf97664a16b011e8a8dd75772b   	1       	Fri Jan 12 11:36:08 2018	DEPLOYED	kube-master-1.9.0-kubernikus.0	kubernikus-michi
```

Reply:
I can do that, yes